templates/front-deployment.yaml

```diff
@@ -31,6 +31,12 @@ spec:
               value: {{ .Values.apiUrl | quote }}
             - name: LAGO_DISABLE_SIGNUP
               value: {{ not .Values.global.signup.enabled | quote }}
+            {{- with .Values.front.extraEnv }}
```

Comment:
I think its a good addition! 
But it still better to have clear config values (eg: `disableWalletRefresh`) since an env var can only apply on one service, and it's also easier for users to understand what happen instead of reading the doc to find the right env var

Reply:
Sure we will see that later :